andela/team-odd-bn-backend
# #171177795 Return fully info about who liked or disliked

#### What does this PR do?
 - Displays more detailed result which includes who liked or dislike
#### Description of Task to be completed?
-  Join booking table and like table
- Display more about who liked or dislike
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-backend.git`
- `cd team-odd-bn-backend`
- git checkout ch-user-like-dislike-171177795
- `npm install`
- Make sure your `.env` file is updated.
- Drop existing migration `npm run dropTables`
- Update migration `npm run db-migrate`
- Populate the seed data `npm run db-seed-dev`
- `npm run dev-start` check API in development 
> Create booking `localhost:3000/api/v1/accommodations/{tripId}/likes`.

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171177795](https://www.pivotaltracker.com/story/show/171177795)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40672143/74914344-f1982280-53ca-11ea-8a1f-9275d06ff309.png)


#### Questions:
N/A
